app/models/pickup_checklist.rb

```diff
@@ -2,6 +2,6 @@ class PickupChecklist < SimpleDelegator
   delegate :zipcode, to: :zone
 
   def donations
-    super.confirmed
+    super.confirmed.order(created_at: :asc)
```

Comment:
If you're at all worried about another order creeping in, I've started using AR's `.reorder` method here, which wipes out any other ordering and ensures that your started order trumps any others.